modules/calib3d/src/fundam.cpp

```diff
@@ -909,6 +909,14 @@ void cv::computeCorrespondEpilines( InputArray _points, int whichImage,
     }
 }
 
+static inline double scaleFor(double x){
+    return (std::fabs(x) > std::numeric_limits<double>::epsilon()) ? 1./x : 1.;
```

Comment:
Double computations are usually truncated on float epsilon value too.
These "double" results are used for intermediate calculations only (for better precision).
But finally, they are converted into floats in functions which works with images (like `wapr()`).

Reply:
changed to <float>::epsilon()